### DISCUSSION TITLE
Should we merge example site with guides?

### DISCUSSION THREAD

**Discussion by BernatBC** [hugo-themes/toha#930] [General]
I've been thinking for a while on what's the point of maintaining two sites (hugo-toha.github.io and guides). Would it be better to collect all guide posts into hugo-toha.github.io and delete the guides site?

Maybe I'm missing the point of having two separate sites.

What are your thoughts?

**Comment by hossainemruz** [comment]
`hugo-toha.github.io` repo serves as a template for the user to get started easily. This is a minimal repository with example of commonly used features. User can just fork this repository, rename and replace respective data with their own data. They will have a ready to deploy site in a few minutes.
